devp2p/discovery.py

```diff
@@ -218,7 +222,7 @@ def __init__(self, app, transport):
 
     def get_node(self, nodeid, address=None):
         "return node or create new, update address if supplied"
-        assert isinstance(nodeid, str)
+        assert isinstance(nodeid, bytes)
```

Comment:
Nevermind.  This isn't a problem.  I shouldn't review pull requests prior to having my morning ☕